api/nodes/serializers.py

```diff
@@ -26,9 +26,12 @@ def to_internal_value(self, data):
 
 
 class NodeSerializer(JSONAPISerializer):
-    # TODO: If we have to redo this implementation in any of the other serializers, subclass ChoiceField and make it
-    # handle blank choices properly. Currently DRF ChoiceFields ignore blank options, which is incorrect in this
-    # instance
+    """TODO: If we have to redo this implementation in any of the other serializers, subclass ChoiceField and make it
```

Comment:
I don't *think* so, because I think it's just the views that get made into docs, but I'd lean towards keeping it the # comments.